src/main/resources/migration/common/V47__create_pathfinder_tagging_table.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE electronic_monitoring_provider (
                                 id SERIAL NOT NULL CONSTRAINT electronic_monitoring_provider_pk PRIMARY KEY,
                                 licence_id INTEGER NOT NULL CONSTRAINT electronic_monitoring_provider_licence_fk REFERENCES licence(id) UNIQUE,
                                 is_to_be_tagged_for_programme BOOLEAN,
-                                programme_name VARCHAR(100),
+                                programme_name TEXT,
                                 date_created TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
```

Comment:
The check sum might fail, you might have to do a separate sql file when you update the column type

```
ALTER TABLE electronic_monitoring_provider
               ALTER COLUMN programme_name TYPE TEXT;
```

unless this as not be deployed to dev yet!